planetiler-core/src/main/java/com/onthegomap/planetiler/geo/TileCoord.java

```diff
@@ -118,6 +122,17 @@ public Coordinate getLatLon() {
     );
   }
 
+  /** Returns the latitude/longitude of the northwest corner of this tile. */
+  public Geometry getEnvelope() {
+    double worldWidthAtZoom = Math.pow(2, z);
+    return JTS_FACTORY.toGeometry(new Envelope(
+      GeoUtils.getWorldLon(x / worldWidthAtZoom),
+      GeoUtils.getWorldLon((x + 1) / worldWidthAtZoom),
+      GeoUtils.getWorldLat(y / worldWidthAtZoom),
+      GeoUtils.getWorldLat((y + 1) / worldWidthAtZoom)
+    ));
+  }
```

Comment:
this will need to change based on re-projecting the geometry we test against, but let's make sure to add 1 test to TileCoordTests to make sure it does what we expect